text/0101-revamping-the-sdk-performance-api.md

```diff
@@ -0,0 +1,273 @@
+| `span_id`         | Span ID of the span                                          | String          | Format is identical between Sentry and OpenTelemetry                                                                                                                                                                                                               |
+| `parent_span_id`  | Parent span ID of the span                                   | String          | Format is identical between Sentry and OpenTelemetry. If empty this is a root span (transaction).                                                                                                                                                                  |
+| `end_timestamp`   | Timestamp when span was finished                             | String \| Float |                                                                                                                                                                                                                                                                    |
+| `start_timestamp` | Timestamp when span was finished                             | String \| Float |                                                                                                                                                                                                                                                                    |
+| `op`              | Operation type of the span                                   | String          | Use is discouraged but kept for backwards compatibility for product features                                                                                                                                                                                       |
+| `status`          | Status of the span                                           | String          | An optional final status for this span. Can have three possible values: 'ok', 'error', 'unset'. Same as OpenTelemetry's Span Status                                                                                                                                |
+| `attributes`      | A set of attributes on the span.                             | Object          | This maps to `span.data` in the current schema for spans. There is no existing mapping for this in the current transaction schema. The keys of attributes are well known values, and defined by a combination of OpenTelemtry's and Sentry's semantic conventions. |
+| `measurements`    | Measurements which holds observed values such as web vitals. | Object          |                                                                                                                                                                                                                                                                    |
```

Comment:
Is this a good time to structure the data further and define what `Object` looks like for each field? The more we structure the data, the better product experience we can provide on the server side.